tmt/steps/report/reportportal.py

```diff
@@ -318,7 +318,10 @@ def go(self) -> None:
         launch_attributes = self.construct_launch_attributes(suite_per_plan, attributes)
 
         launch_description = self.data.launch_description or self.step.plan.summary
-
+        # Checks whether Testing Farm provided link to artifacts
```

Comment:
I wouldn't tie this to TF in this way - the motivation started in the TF environment, but the concept is generic. The variable can also be set locally, so "Check whether artifact URL has been provided" would serve us better in the long run. One day, tmt might be used by other CI systems directly, who knows?

The variable deserves its paragraph in docs, BTW, somewhere under https://github.com/teemtee/tmt/blob/main/docs/overview.rst?plain=1#L346.

Reply:
I already created a JIRA issue for myself to do it, but feel free to create an issue here as well. Don't think I'll make it in time for 1.32 as it is supposed to be released this week.

Reply:
Yeah, probably too late for 1.32 given the number of PRs still in the queue :/